src/Domains/Guild/Customers/Actions/CreatePeopleAction.php

```diff
@@ -115,10 +116,26 @@ public function execute(): People
                 }
             }
 
+            if ($this->peopleData->peopleEmploymentHistory) {
+                foreach ($this->peopleData->peopleEmploymentHistory as $employmentHistory) {
+                    $people->employmentHistory()->save(new PeopleEmploymentHistory([
+                        'organizations_id' => $employmentHistory['organizations_id'],
+                        'apps_id' => $this->peopleData->app->getId(),
+                        'position' => $employmentHistory['position'],
+                        'income' => $employmentHistory['income'],
+                        'start_date' => $employmentHistory['start_date'],
+                        'end_date' => $employmentHistory['end_date'],
+                        'status' => $employmentHistory['status'],
+                        'income_type' => $employmentHistory['income_type'] ?? null,
+                    ]));
+                }
+            }
+
             if (! empty($addressesToAdd)) {
                 $people->address()->saveMany($addressesToAdd);
             }
         }
+        $people->refresh();
```

Comment:
@FredPeal dont think we need the refresh on the create no?

Reply:
Is the correct way when you add a relationship

Reply:
I think , I dont know xD